static/js/lib/fetch.js

```diff
@@ -0,0 +1,381 @@
+(function() {
```

Comment:
Cool - thanks - hopefully at some point in the future we will be able to use fetch :)